public/app/core/utils/explore.test.ts

```diff
@@ -8,23 +8,12 @@ import {
   hasNonEmptyQuery,
```

Comment:
Test for sorting were moved to `grafana/data`. Removing unused imports and tests.